cmd/truss/main.go

```diff
@@ -325,33 +335,30 @@ func readPreviousGeneration(serviceDir string) ([]truss.NamedReadWriter, error)
 }
 
 // simpleFileConstructor has function makeSimpleFile of type filepath.WalkFunc
-// This allows for filepath.Walk to be called with makeSimpleFile and build a truss.SimpleFile
-// for all files in a direcotry
+// This allows for filepath.Walk to be called with makeSimpleFile and build a fileSystem
+// for all files in a directory
 type simpleFileConstructor struct {
```

Comment:
I know this change wasn't introduced here, but seeing the word "constructor" in Go code kind of sets off my smell alarm. I know it doesn't really imply anything here, but since it doesn't, it shouldn't use a word that carries all that baggage with it.